src/FM.LiveSwitch.Hammer/ScanTestMediaServer.cs

```diff
@@ -78,6 +78,10 @@ public async Task<ScanTestMediaServerResult> Run(MediaServerInfo mediaServer, Ca
                                     // pass
                                     result.SetScenarioResult(scenario, ScanTestScenarioResult.Pass(scenario, certificateValidFor));
                                 }
+                                catch (NoIceServersException)
+                                {
+                                    result.SetScenarioResult(scenario, ScanTestScenarioResult.Skip(scenario, "Skipped due to no ice servers."));
```

Comment:
Suggested wording:

`"No ICE servers."`

"ICE" as an acronym should be capitalized "Skipped due to" is redundant -- the scenario result is already "skipped" and the message is the reason.